### PR TITLE
Fix - #27294 content updated

### LINKED ISSUE #27294: Hello World - conditional content update - Making and committing changes section
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Hello World](https://docs.github.com/en/get-started/quickstart/hello-world)

### What part(s) of the article would you like to see updated?

# Background
[Making and committing changes](https://docs.github.com/en/get-started/quickstart/hello-world#making-and-committing-changes) is different than what I experience (in my browser).

# Steps to reproduce
What I see when I view  [Hello World](https://docs.github.com/en/get-started/quickstart/hello-world) quickstart tutorial (experience/content is same in Chrome and Safari; whether I am logged in to GH in browser or not):
image

When I follow along tutorial, and to [Making and committing changes](https://docs.github.com/en/get-started/quickstart/hello-world#making-and-committing-changes) section, I see this "Commit changes" box (not "Propose changes" box).
image

# Expected Behavior
I expect the instructions to reflect "Commit changes" box (and button) language.

# Why
This document should be updated to reflect actual(/current) user experience and to reduce confusion over differing language which will lead to improve user satisfaction when learning GH.

### Additional information

Rationale for submitting (why am I doing this): I am getting back into coding and am (re-learning) GH and all the goodness around the workflows and interacting with the system. 

I am going thru the process of submitting the document improvement (a VERY minor tweak, IMO) to understand the PR process and how to contribute as a team member in a larger developer pond.

I did review the [existing open Issues](https://github.com/github/docs/issues), and did not see one that was related to this new issue. I plan on submitting a Pull Request shortly after this issue is submitted.

[maintainer edit]
### Content plan
 You, or anyone else, are welcome to submit a PR, updating the directions from "Propose changes" in steps 5 and 6 to "Commit changes".

### PR DESCRIPTION
**Pull request by Shikha1234**
**Issue:**
#27294 Making and committing changes is different from the tutorial experience

**Steps to Reproduce:**

Navigate to the Hello World quickstart tutorial.
Observe the "Commit changes" box instead of the expected "Propose changes" box in the "Making and committing changes" section.
**Description:**

This pull request addresses the issue with the tutorial where the "Commit changes" box is displayed instead of the expected "Propose changes" box in the "Making and committing changes" section. The issue has been consistently observed across different browsers (Chrome and Safari) and whether logged in to GitHub in the browser or not.

**Changes Made:**
I have updated the tutorial content to ensure that the correct terminology, "Propose changes," is displayed in the "Making and committing changes" section, aligning it with the expected tutorial experience.

Closes: https://github.com/github/docs/issues/27294

Checklist:

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Edited the content from "Propose Changes " to "Commit Changes" in "Making and committing changes
"  heading

### Check off the following:

[Y] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [Y] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@Shikha1234 Thanks for submitting this PR! I've edited your post to [link your PR to your issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

I'll get this triaged for review! :zap:

File: content/get-started/quickstart/hello-world.md [hunk 1]
```diff
@@ -89,8 +89,8 @@ You can make and save changes to the files in your repository. On {% data variab
 1. In the editor, write a bit about yourself. Try using different Markdown elements.
 {% ifversion code-search-code-view %}
```
Comment (isaacmbrown) [inline content/get-started/quickstart/hello-world.md:90]: Hi @Shikha1234, I think we can simplify the versioning here to make it easier for people to see the logic

Can you modify lines 90-97 to the following?

```
{% ifversion code-search-code-view %}1. Click **Commit changes...**.{% endif %}
1. In the "Commit changes" box, write a commit message that describes your changes.
1. Click **Commit changes**.
```

**Comment by cmwilson21** [comment]
👋 @Shikha1234 Just checking in! Have you had a chance to review `@isaacmbrown`'s suggested changes?

File: content/get-started/quickstart/hello-world.md (hunk 1 above)
Comment (callumjohnspiller) [reply content/get-started/quickstart/hello-world.md:90]: Is it the case that the lines:
`1. In the "Propose changes" box, write a commit message that describes your changes.
1. Click **Propose changes**.`
were left in erroneously - I noticed this issue too and glad to see someone was working on a fix! I was struggling to see why there were two instances of the same message!

**Comment by Shikha1234** [comment]
Hi Team, I have updated the changes mentioned in the comments. Please review  and Let me know if any modifications required

File: content/get-started/quickstart/hello-world.md (hunk 1 above)
Comment (isaacmbrown) [reply content/get-started/quickstart/hello-world.md]: @callumjohnspiller, that's right, probably we changed the UI text to "Propose changes" at a certain point in the code search feature's development, then changed it back to "Commit changes" later

**Review by isaacmbrown — APPROVED** [review]
Thanks @Shikha1234! 🚀

**Pull request merged**